src/installer/installer.go

```diff
@@ -116,6 +116,11 @@ func (i *installer) InstallNode() error {
 	if err = i.writeImageToDisk(ignitionPath); err != nil {
 		return err
 	}
+
+	if err = i.ops.SetBootOrder(i.Device); err != nil {
+		return err
```

Comment:
We are always doing this. Not only in the scope of metal3 boot.
In case of boot-it-yourself we should definitely not exit if there was an error in changing boot device.
In some cases the HW does not support it

Reply:
Very good point.

Reply:
There is a check for efi inside the function.
`_, err := o.ExecPrivilegeCommand(o.logWriter, "test", "-d", "/sys/firmware/efi")`

Reply:
While this should do the trick I think that the SetBootOrder should not fail the installation in case of failure (at least not at first).
I suggest that we just log a warning in case we fail here, so we can get the logic merged and not worry about failing installation due to this change.
After this will run for a couple of weeks and we can check the logs to see how it behaves on customers installations we can add the `return err`

Reply:
Done